### DISCUSSION TITLE
v0.11.15

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#588] [Releases]
## What's Changed
* [Bug] added back deleted telegram notification class by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/586
* Release v0.11.15 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/587

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.11.14...v0.11.15

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.11.15'>v0.11.15</a>.</em>

**Comment by corasaniti** [comment]
Thanks
